app/assets/stylesheets/magnific-popup.css

```diff
@@ -0,0 +1,351 @@
+/* Magnific Popup CSS */
```

Comment:
Toàn bộ thư viện của template em lấy về đặt trong folder `vendor/` cấu trúc thư mục thì giống như trong folder `assets` nhé